codegen/src/main/scala/caliban/codegen/Codegen.scala

```diff
@@ -7,30 +7,30 @@ import caliban.parsing.adt.Document
 import caliban.parsing.Parser
 
 object Codegen {
-  def getSchema(path: String): Task[Document] =
-    if (path.startsWith("http")) {
-      IntrospectionClient.introspect(path)
-    } else {
-      Task(scala.io.Source.fromFile(path))
-        .bracket(f => UIO(f.close()), f => Task(f.mkString))
-        .flatMap(Parser.parseQuery)
-    }
-
   def generate(
-    schemaPath: String,
-    toPath: String,
-    fmtPath: Option[String],
+    arguments: CommandLineArguments,
```

Comment:
How about calling this something less specific like `Config` or `Options`? Since `caliban-codegen` could be used outside of sbt.

Reply:
Fair point, I'll change that 👍